controller/array_action/array_mediator_svc.py

```diff
@@ -195,19 +196,28 @@ def _generate_snapshot_response(self, cli_snapshot, source_volume_name):
                         is_ready=True,
                         array_type=self.array_type)
 
+    def _generate_snapshot_response_with_verification(self, cli_object):
+        if not cli_object.FC_id:
+            logger.error("FlashCopy Mapping not found for target volume: {}".format(cli_object.name))
+            raise controller_errors.ExpectedSnapshotButFoundVolumeError(cli_object.name, self.endpoint)
+        fcmap = self._get_fcmap_as_target_if_exists(cli_object.name)
+        if fcmap is None or fcmap.copy_rate != 0:
```

Comment:
are you sure it's not `'0'`?

Reply:
either way, please cover all exit scenarios of this function with unit tests

Reply:
it is a string but it fixed on the mediator work..

Reply:
no svc UT on this scope

Reply:
I don't see the scenarios covered in the mediator work - would you like to open a conversation where it's covered?

Reply:
when we finish the rest I will take a look on the coverage of the tests, dont resolve this until covered

Reply:
[done](https://github.com/IBM/ibm-block-csi-driver/blob/c13ab9f5135966d8de1f1286cef7d7520ed74930/controller/tests/array_action/svc/array_mediator_svc_test.py#L280)

Reply:
I only see one exist scenario (`fcmap.copy_rate != '0'`), could you point me to the rest?

Reply:
its only for generate response.. what more do you need?

Reply:
what about `fcmap is None`, etc...?

Reply:
btw, all other calls are with copy_rate = '0' so it checked too

Reply:
lets do this in the mediator PR?

Reply:
open there plz

Reply:
np